src/main/java/kitEditor/Sample.java

```diff
@@ -104,6 +110,10 @@ public static Sample createFromWav(File file,
         return s;
     }
 
+    public static Sample dupeSample(Sample sample) throws IOException, CloneNotSupportedException {
+      return (Sample)sample.clone();
+    }
```

Comment:
This one confuses me.
Is there any reason why one would call `Sample.dupleSample(sample)` instead of `sample.clone()`?

Reply:
Something to watch out for...
It might be a source of bugs, that Object.clone() only does shallow copy of the object fields.
This is probably not what you want!

Reply:
I will defer to your judgment on this, and as mentioned earlier it would probably be more sensible to use a copy constructor.